### DISCUSSION TITLE
First time using this library, and got error in hello world, please help

### DISCUSSION THREAD

**Discussion by kenjichanhkg** [rnmapbox/maps#1435] [Q&A] [answered]
I init my project by "npx react-native init example".
Then I added this library by "yarn add @react-native-mapbox-gl/maps"
Then I added the import statement at the top:
`   import MapboxGL from "@react-native-mapbox-gl/maps";`

Then I got this error, does anyone know why?

![image](https://user-images.githubusercontent.com/7701160/123994255-c6323880-d9ff-11eb-8b35-4520b9d68607.png)

**Comment by ferdicus** [comment]
hi, did you follow our installation guide? https://github.com/react-native-mapbox-gl/maps/blob/master/ios/install.md

**Reply by kenjichanhkg** [reply]
no, but.....now I cannot even do "arch -x86_64 pod install", I got errors.
Anyone knows why?

I haven't even added this yet:
`pod 'react-native-mapbox-gl', :path => '../node_modules/@react-native-mapbox-gl/maps'`

![image](https://user-images.githubusercontent.com/7701160/123998008-95ec9900-da03-11eb-9787-ac812bb089d9.png)

**Reply by kenjichanhkg** [reply]
if I removed this line in package.json, I could pod install without error. Why is that?
"@react-native-mapbox-gl/maps": "^8.2.1",

**Reply by ferdicus** [reply]
please open a proper issue ticket, this is a discussion.  
make sure to follow the issue template instructions, otherwise we can not help you.

**Reply by kenjichanhkg** [reply]
created an issue for it
https://github.com/react-native-mapbox-gl/maps/issues/1449

**Reply by kenjichanhkg** [reply]
Can someone please look at my issue?

**Comment by ferdicus** [comment, marked as answer]
Answer is in ticket: https://github.com/react-native-mapbox-gl/maps/issues/1449
